packages/migrate/Migration.js

```diff
@@ -202,6 +205,46 @@ class Migration {
     return { interfaceAdapter, resolver, context, deployer };
   }
 
+  async deployAndLinkLogger(options, resolver) {
+    const {
+      networks,
+      network,
+      network_id,
+      provider,
+      solidityLogging
+    } = options;
+    if (solidityLogging) {
+      let Console;
+      try {
+        Console = resolver.require("Console");
+      } catch (error) {
+        return;
```

Comment:
Is it okay that we just let this fail?

Reply:
I suppose we probably should let this fail, or at the very least warn them? It seems better since they have the logging turned on and it would fail.

Reply:
It might be the right behavior to abort like this. I'm just wondering if there's a way we can detect certain allowed failure conditions, rather than catch-all aborting every kind of exception with no notice. Like, surely, in some situations where things break, we should alert the user, no?

Reply:
So really this code should never throw unless there is an error on our side when a user is using Truffle, right? I suppose if they are using the migration library stand-alone and don't configure something correctly it might fail as well. I might be in favor of just printing a warning to the screen that "you are using the solidity logging flag but we cannot resolve the library" or something like that.

Reply:
So this function got deleted and it now gets deployed and linked like all of the other testing libraries in  `packages/core/lib/testing/SolidityTest.js`.